### DISCUSSION TITLE
Ability to move and delete photos

### DISCUSSION THREAD

**Discussion by ChristoffM** [photoview/photoview#824] [Ideas]
Are there ways to use PhotoView **to delete and move photos?** It seems it is only a viewer (as the name suggests)?

Are there perhaps plans to create the ability to delete and move photos from within the app? Could this be a feature request? Maybe combined with #801 ?

By the way, thanks for a great application! I wrote a basic similar app a few years ago for very much the same reasons, and am so happy to have found this much more mature and refined project to replace my own one.

**Comment by viktorstrate** [comment]
Thank you for the kind words 😊

Modifying the photos from Photoview sidesteps the aim of the project.
The documentation describes why here: https://photoview.github.io/en/docs/#aim-and-values
